src/objects.rs

```diff
@@ -1286,6 +1286,30 @@ impl GetObjectType for StopPoint {
     }
 }
 
+impl From<&StopArea> for StopPoint {
+    fn from(stop_area: &StopArea) -> Self {
+        StopPoint {
+            id: format!("VSP:{}", stop_area.id),
+            name: stop_area.name.clone(),
+            visible: stop_area.visible,
+            coord: stop_area.coord,
+            codes: KeysValues::default(),
+            object_properties: PropertiesMap::default(),
+            comment_links: LinksT::default(),
+            code: None,
+            stop_area_id: stop_area.id.clone(),
+            timezone: stop_area.timezone,
+            geometry_id: stop_area.geometry_id.clone(),
```

Comment:
Maybe set to None to avoid problems.
Such as a polygon geometry for a StopArea that don't exist for a StopPoint (only POINT is allowed).

```suggestion
            geometry_id: None,
```